rules/windows/command_and_control_encrypted_channel_freesslcert.toml

```diff
@@ -23,12 +23,12 @@ If enabling an EQL rule on a non-elastic-agent index (such as beats) for version
 risk_score = 21
 rule_id = "e3cf38fa-d5b8-46cc-87f9-4a7513e4281d"
 severity = "low"
-tags = ["Elastic", "Host", "Windows", "Threat Detection", "Command and Control"]
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Command and Control", "Elastic Endgame"]
 timestamp_override = "event.ingested"
 type = "eql"
 
 query = '''
-network where network.protocol == "dns" and
+network where (network.protocol == "dns" or event.action == "request_event") and
```

Comment:
Since `network.protocol` doesn't exist in endgame, this would force that field to be optional.

Reply:
I don't think it needs to be optional here, the field exists in the mapping, so no errors will be thrown
![image](https://user-images.githubusercontent.com/26856693/193352268-bf0ef4a6-7680-4158-b674-e6b32a069313.png)

Reply:
If it's correct, then the schema mapping provided needs to be updated.

Reply:
> Based on Endgame ECS Mapping [Excel](https://docs.google.com/spreadsheets/d/1HpWZ8OAvMa-HRAMX4NQmQ4UYo4Uks0bZRpBqMSICx2w/edit#gid=2031619638) doc, the network.protocol field is not supported. This means the original data may provide it, but there is no ECS conversion.

I'm using the actual mapping used by the index as basis, it has fields that it doesn't use, which is the case for this one.